Core.ApplicationServices/Organizations/IOrganizationUnitService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Core.Abstractions.Types;
+using Core.ApplicationServices.Model.Organizations;
+using Core.DomainModel.Organization;
+
+namespace Core.ApplicationServices.Organizations
+{
+    public interface IOrganizationUnitService
+    {
+        Result<OrganizationRegistrationDetails, OperationError> GetOrganizationRegistrations(int organizationId, int unitId);
+        Maybe<OperationError> DeleteSelectedOrganizationRegistrations(int organizationId, int unitId, OrganizationRegistrationChangeParameters parameters);
+        Maybe<OperationError> DeleteAllUnitOrganizationRegistrations(int organizationId, int unitId);
```

Comment:
this one will be DeleteOrganizationRegistrations but without the additional parameter making the other one an overload